symfony/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "require": {
+        "php": "^7.3",
```

Comment:
Need to discuss. (Doctrine 3.x requires PHP 7.3, https://packagist.org/packages/doctrine/orm#dev-master. But Doctrine 3 not yet added since 3.x under development)
Migration guide to Doctrine 3 - https://github.com/doctrine/migrations/blob/3.0.x/UPGRADE.md

Reply:
Use Doctrine 3.x